source/MRMesh/MRFillContours2D.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include "MRMeshFwd.h"
+
+namespace MR
+{
+
+MRMESH_API bool fillContours2D( Mesh& mesh, std::vector<EdgePath>& paths );
```

Comment:
please add a comment what the function does, and what should be in the arguments

Reply:
fixed